client/src/Components/Origin.jsx

```diff
@@ -22,25 +23,61 @@ function Origin (props) {
 
       return b[countryNameB] - a[countryNameA];
     })
+
+    sortedCountries.forEach(country => {
+      country[Object.keys(country)] = Math.floor(Object.values(country)[0] * props.backers);
+    })
     setCountries(sortedCountries);
   };
 
+  const getTopCities = (locations) => {
+    let allCities = [];
+
+    locations.forEach(location => {
+      for (let country in location) {
+        let cities = location[country].cities, countryBackers = location[country].chance * props.backers;
+
+        cities.forEach(city => {
+          let cityName = Object.keys(city)[0], cityBackers = Math.floor(city[cityName] * countryBackers);
+
+          allCities.push({[cityName]: cityBackers, 'country': country});
+        })
+      }
+    })
+
+    let sortedCities = allCities.sort((a, b) => {
+      let cityNameA = [Object.keys(a)[0]], cityNameB = [Object.keys(b)[0]];
+
```

Comment:
why is there a blank line here? it seems logical in other spots where you're showing that a variable is separate from the previous variable, but this one seems unnecessary.